### DISCUSSION TITLE
VTDecompressionSessionDecodeFrame fails with OSStatus=-12909

### DISCUSSION THREAD

**Discussion by martinzsapka** [HaishinKit/SRTHaishinKit.swift#56] [Q&A]
Hello!

I'm trying to receive an h264 srt stream, but the decoder fails with error OSStatus=-12909. Specifically, the VTDecompressionSessionDecodeFrame call. Of course because of that, I don't get any imagebuffer.

Do you have any guesses as to what I may have misplaced? Maybe some configuration is missing?

Thanks!

**Comment by shogo4405** [comment]
No way. It's just a bug. I will find some time to fix it.

**Reply by kazhuravlev** [reply]
hi! Any changes?
